data/src/main/java/com/netflix/iceberg/data/TableScanIterable.java

```diff
@@ -66,14 +67,16 @@
     this.ops = ((HasTableOperations) scan.table()).operations();
     this.projection = scan.schema();
     this.reuseContainers = reuseContainers;
+    this.caseSensitive = scan.isCaseSensitive();
+
```

Comment:
Nit: unnecessary blank line addition (leads to unnecessary blank line removal and unnecessary commit conflicts).

Reply:
Ack.